website/content/en/highlights/2022-11-07-0-26-0-upgrade-guide.md

```diff
@@ -14,6 +14,7 @@ Vector's 0.26.0 release includes **breaking changes**:
 1. [Removal of `check_fields` condition type](#check-fields-removal)
 2. [Removal of `lanes` parameter for `route` transform](#lanes-parameter-route-transform-removal)
 3. [Removal of `geoip` transform](#geo-transform-removal)
+4. [Changes to `native` Protobuf codec](#vector-native-protobuf)
```

Comment:
I imagine we should make simultaneous changes to the native-json codec? (as part of the same release)